### PR TITLE
Ft get specific party endpoint 163587441

### PR DESCRIPTION
**Pull request by iamcristos**
#### What does this PR do?
Gets a specific political party

#### Description of Task to be completed?
/GET '/api/v1/parties/:id' 
 
#### How should this be manually tested?
After cloning the repo run npm then input '/api/v1/parties/:id' as your url on postman

#### What are the relevant pivotal tracker stories?
#163587441
#### Questions:

### CODE REVIEW

File: src/middlewares/partyMiddleware.js [hunk 1]
```diff
@@ -16,6 +16,24 @@ class partyValidator {
...
+        console.log(item);
+      if (item.id === Id ) {
+          return res.send(item)
+        // return res.status(404).send({
+        //   success: true,
+        //   message: 'Party dont exist'
+        // });
+      } else {
+        next()
+      }
```
Comment (FredDominant) [inline src/middlewares/partyMiddleware.js:30]: Please remove the commented code

File: test/party.test.js [hunk 2]
```diff
@@ -41,4 +43,16 @@ describe('/GET ALL PARTIES', ()=>{
       })
       .end(done)
   })
-})
+});
+
+// describe('/GET A PARTY', ()=>{
```
Comment (FredDominant) [inline test/party.test.js]: Here also

**Review by FredDominant — CHANGES_REQUESTED** [review]

File: src/middlewares/partyMiddleware.js (hunk 1 above)
Comment (iamcristos) [reply src/middlewares/partyMiddleware.js:30]: Aiit tnx

File: test/party.test.js (hunk 2 above)
Comment (iamcristos) [reply test/party.test.js]: ohk

**Pull request merged**